src/Errors/Errors.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import '../Errors/Errors.css'
+import { NavLink } from "react-router-dom/cjs/react-router-dom.min";
+
+const Errors = ({ errorMessage }) => {
```

Comment:
I think this component can be refactored to be more concise, since the two returns are so similar. Not sure how yet, but something to consider when we refactor.